File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/creatingalanguageservice(managedpackageframework)/cs/mysource.cs
//<Snippet4>
using Microsoft.VisualStudio;
using Microsoft.VisualStudio.Package;
using Microsoft.VisualStudio.TextManager.Interop;
using Microsoft.VisualStudio.OLE.Interop;
//</Snippet4>

//<Snippet5>
namespace MyLanguagePackage
{
    class MySource : Source
    {
        public MySource(LanguageService service,
                        IVsTextLines textLines,
                        Colorizer colorizer)
            : base(service, textLines, colorizer)
        {
        }
    }
}
//</Snippet5>

File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/creatingalanguageservice(managedpackageframework)/vb/mysource.vb
'Copyright (c) Microsoft Corporation. All rights reserved.
'<Snippet4>
Imports Microsoft.VisualStudio
Imports Microsoft.VisualStudio.Package
Imports Microsoft.VisualStudio.TextManager.Interop
Imports Microsoft.VisualStudio.OLE.Interop
'</Snippet4>

'<Snippet5>
Namespace MyLanguagePackage
	Friend Class MySource
		Inherits Source
        Public Sub New(ByVal service As LanguageService, ByVal textLines As IVsTextLines,
                       ByVal colorizer As Colorizer)
            MyBase.New(service, textLines, colorizer)
        End Sub
	End Class
End Namespace
'</Snippet5>